tests/push/test_bulk_push_rule_evaluator.py

```diff
@@ -370,3 +370,43 @@ def test_room_mentions(self) -> None:
                 },
             )
         )
+
+    @override_config({"experimental_features": {"msc3958_supress_edit_notifs": True}})
+    def test_suppress_edits(self) -> None:
+        """Event edits should not generate notifications."""
```

Comment:
Nitpicking, but maybe:

```suggestion
"""Under the default push rules, event edits should not generate notifications."""
```

since, as the MSC says, this new push rule can be disabled if the user did want to be notified about edits.

(Although you're never normally notified about your own actions, and only you can edit your messages. Maybe this is redundant? :shrug:)